CMakeLists.txt

```diff
@@ -132,34 +132,34 @@ set(f2py_dir ${CMAKE_SOURCE_DIR}/src/f2py)
 configure_file(${F2PY_INCLUDE_DIR}/fortranobject.c fortranobject.c COPYONLY)
 set(f2py_source fortranobject.c)
 
-set(chromo_logging chromo_openlogfile chromo_closelogfile)
+set(chromo_functions chromo_openlogfile chromo_closelogfile npyrng)
 set(logging_source ${fortran_dir}/logging.f)
-set(rangen_source ${fortran_dir}/rangen.fpp)
+set(rangen_source ${fortran_dir}/rangen.fpp ${fortran_dir}/rangen.c ${fortran_dir}/normal.c)
 
 
 ### eposlhc
-file(GLOB eposlhc_sources ${fortran_dir}/EPOS-LHC/sources/*.f)
+file(GLOB eposlhc_sources ${fortran_dir}/epos/sources/*.f)
 list(FILTER eposlhc_sources EXCLUDE REGEX epos_example\.f)
 
 
 f2py_add_module(_eposlhc
   FUNCTIONS
   aaset ainit aepos afinal hepmcstore
   getcharge idtrafo initepos initeposevt
-  xsection init_rmmard ${chromo_logging}
+  xsection rmmard ${chromo_functions}
   SOURCES
   ${eposlhc_sources}
   ${logging_source}
   ${rangen_source}
   INTERFACE_SOURCES
-  ${fortran_dir}/EPOS-LHC/sources/epos-bas-lhc.f 
-  ${fortran_dir}/EPOS-LHC/sources/epos-ids-lhc.f 
-  ${fortran_dir}/EPOS-LHC/sources/epos_interface.f
-  ${fortran_dir}/EPOS-LHC/sources/epos-random.f
+  ${fortran_dir}/epos/sources/epos-bas-lhc.f
+  ${fortran_dir}/epos/sources/epos-ids-lhc.f
+  ${fortran_dir}/epos/sources/epos_interface.f
+  ${fortran_dir}/epos/sources/epos-random.f
   ${rangen_source}
   ${logging_source}
   INCLUDE_DIRS
-  ${fortran_dir}/EPOS-LHC/sources
+  ${fortran_dir}/epos/sources ${NUMPY_INCLUDE_DIRS}
```

Comment:
I cannot find NUMPY_INCLUDE_DIRS variable. Where is it defined?

Reply:
In line 50, where it was before?